src/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+
+const LinkView = ({ targetTo, text, path }) =>
+  (<a
+    tabIndex="0"
+    role="link"
```

Comment:
It's a WAI-ARIA attribute. I added role (and tabIndex) as [eslint no-static-element-interactions rule](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md) was triggered.